src/rpc/rpc_benchmark.js

```diff
@@ -110,14 +117,15 @@ schema.compile();
 
 // create rpc
 const rpc = new RPC({
-    schema: schema,
+    client_factory: api.client_factory_from_schema(schema),
```

Comment:
Hmm, I want to make sure `rpc` module is usable even without `api` module.
Do you think this can be done?

Reply:
Ok we have it now in a sepetate PR: https://github.com/noobaa/noobaa-core/pull/6178
We can wait to merge this one untill the other one is merged

Reply:
Rebased against `master` and removed dependency with the `api` module